From 68d6654042c64465247c26ea95d3bd5d397a5db8 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 20 Apr 2021 17:05:25 +0300
Subject: [PATCH 1487/1921] arm64: dts: marvell: add pin control settings to
 i2c1

Add pin control settings to cp0_i2c1 on CN9130-CRB platforms.
When pin control entries are missing, the following error is
show during boot:

mv64xxx_i2c f2701100.i2c: can't get pinctrl, bus recovery not supported

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: If8be4bc63dba36c027927cb2c644ff5e81773ba5
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50248
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index b400a7b6d64d..45eab1b63d7b 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -145,6 +145,8 @@
 
 &cp0_i2c1 {
 	status = "okay";
+	pinctrl-names = "default";
+	pinctrl-0 = <&cp0_i2c1_pins>;
 	clock-frequency = <100000>;
 };
 
-- 
2.31.1

